diff --git a/tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc b/tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc
index cb603ba7c6c..4b7ba635b58 100644
--- a/tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc
+++ b/tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc
@@ -410,11 +410,7 @@ class DynamicPartitionOpGPU : public AsyncOpKernel {
                                             num_partitions_);

 #if GOOGLE_CUDA
-#if THRUST_VERSION >= 200802
     thrust::constant_iterator<int32> values_in(1);
-#else
-    cub::ConstantInputIterator<int32> values_in(1);
-#endif
 #elif TENSORFLOW_USE_ROCM
     using ConstantInputIterator =
         ::rocprim::constant_iterator<int32, ptrdiff_t>;

